src/Drivers/ImageDriver.php

```diff
@@ -86,6 +87,19 @@ public function flip(FlipDirection $flip): static;
 
     public function pixelate(int $pixelate): static;
 
+    public function watermark(
```

Comment:
Could we also add an `$alpha` parameter that control the translucency of the watermark?